src/config/test_config.c

```diff
@@ -113,7 +113,7 @@ static const WT_CONFIG_CHECK confchk_burst_inserts[] = {
   {"workload_tracking", "category", NULL, NULL, confchk_workload_tracking_subconfigs, 2},
   {NULL, NULL, NULL, NULL, NULL, 0}};
 
-static const WT_CONFIG_CHECK confchk_example_test[] = {
```

Comment:
The git diffs make this a bit confusing to read. Have we just removed the `confchk_base_test` config and added it as `confchk_test_template` in a different location?

Reply:
This is generated with `s_all`. Configurations have been shuffled as I had to sort them in `test_data.py` since two tests were renamed.